papas/simulator.py

```diff
@@ -137,13 +139,30 @@ def simulate_electron(self, ptc):
     def simulate_neutrino(self, ptc):
         self.propagate(ptc)
         
-    def simulate_hadron(self, ptc):
+    def simulate_hadron(self, ptc, enable_matter_scattering):
```

Comment:
Indeed, one should remove this parameter, and your idea to modify the material to get events without scattering is the most elegant! the user can do that if needed, e.g. modifying the beam pipe of the CMS detector in the configuration file of heppy. thanks